tests/test_add_histograms.py

```diff
@@ -495,7 +495,9 @@ def simple_2D(tmp_path):
     for i in range(len(data2)):
         for j in range(len(data2[0])):
             h2.Fill(i, j, data2[i][j])
-    outHistFile = ROOT.TFile.Open(Path(tmp_path) / "file2dim2.root", "UPDATE")
+    outHistFile = ROOT.TFile.Open(
+        "/Users/zobil/Documents/hepconvert/tests/samples/file2dim2.root", "RECREATE"
```

Comment:
@zbilodea Paths like these mean tests will never pass on any machine that isn't yours. Instead use [`pytest`'s `tmp_path` fixture](https://docs.pytest.org/en/latest/how-to/tmp_path.html).